src/Module 1/Module1.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Module_1;
+
+public class Module1
+{
+    static readonly HttpClient client = new HttpClient();
+    static string writePath = @"D:rates.json";
+    static string url = "https://www.nbrb.by/api/exrates/rates/usd?parammode=2&ondate=";
+    public static async Task GetJson()
+    {
+        DateTime datatime = new DateTime(DateTime.Now.Year, 1, 1);        
+
+        try
+        {
+            var response = new HttpResponseMessage[DateTime.Now.Subtract(datatime).Days + 1];
```

Comment:
nit: 
```suggestion
            var responses = new HttpResponseMessage[DateTime.Now.Subtract(datatime).Days + 1];
```

Reply:
но я не очень понимаю зачем тут массив.. ведь после цикла этот массив нигде не используется..

давайте так:
```suggestion
            for (int i = 0; i < DateTime.Now.Subtract(datatime).Days + 1; i++)
            {
                DateTime tempDateTime = datatime.AddDays(i);
                string tempDateTimeToString = $"{tempDateTime.Year}-{tempDateTime.Month}-{tempDateTime.Day}";
                var response = await client.GetAsync($"{url}{tempDateTimeToString}");
                response.EnsureSuccessStatusCode();
                string responseBody = await response.Content.ReadAsStringAsync();
                if (i == 0) DeleteIfExistsJson();
                await AppendToJson(responseBody);
            }
```